schematic/__main__.py

```diff
@@ -24,6 +25,7 @@
 # invoke_without_command=True -> forces the application not to show aids before losing them with a --h
 @click.group(context_settings=CONTEXT_SETTINGS, invoke_without_command=True)
 @click_log.simple_verbosity_option(logger)
+@click.version_option(version=__version__, prog_name="schematic")
```

Comment:
Double check that this works in a docker container. I ran into issues using `__version__` where on my local machine it worked fine, but in docker it would crash because the package was not found.

Reply:
For this I had just used `docker build .` and did a `docker run` on the built image. If it starts up we would just want to confirm that we could print the version to console.

I had tried the same things you did here:

https://github.com/Sage-Bionetworks/schematic/pull/1527/commits/a096639da83a4024d8f3c3874e5d9f6a1c91f279

https://github.com/Sage-Bionetworks/schematic/pull/1527/commits/2e3b3cf594d3d00d858ab4a95603935737215de2